src/components/Favourites/FavouritesFull/FavouritesFull.tsx

```diff
@@ -28,7 +28,7 @@ export default function FavouritesFull() {
       </div>
       <div className="flex w-full justify-center">
         <Link href={'/'}>
-          <Button className="my-6 h-14  text-lg font-medium sm:w-[211px]">
+          <Button className="my-6 h-14  text-lg font-medium sm:w-[211px] ">
```

Comment:
Why is there an additional button inside the link that leads to the checkout page?
Maybe it's better to apply these styles to the Link component and get rid of the extra button tag?

Reply:
fixed